index.html

```diff
@@ -284,16 +285,19 @@ <h2 class="header-text">About <br>Myself</h2>
       </div>
     </div>
   </section>
+  <section class="contact-form">
+    <div class="contact-me">
+      <h2>Contact me</h2>
+      <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+    </div>
+    <form action="/my-handling-form-page" method="post">
+      <input class="input-1" type="text" placeholder="Yeremias"  maxlenght="30" required>
+      <input class="input-2" type="email" name="email" placeholder="natajayanj14@gmail.com" maxlength="30" required>
```

Comment:
Good attempt at trying to implement the form action. Unfortunately, when I submit the form, it still is not behaving correctly. Kindly fix this in order to make it possible for clients to easily contact you from your portfolio.

Check out this [article](https://formspree.io/blog/html-contact-form/)

SEE WHAT I GET AFTER FORM SUBMISSION

![error](https://user-images.githubusercontent.com/84930355/220468800-42aaa0cd-1f9c-4f53-897a-a3500eb1e6a8.PNG)